providerBuild.sh

```diff
@@ -3,18 +3,55 @@
 # Provider Setup Script
 
 # Initialize variables with default values or empty
-ACCOUNT_ADDRESS=""
-KEY_PASSWORD=""
-DOMAIN=""
+ACCOUNT_ADDRESS="akash123456789"
+KEY_PASSWORD="blabla"
+DOMAIN="test.europlots.com"
 NODE=""  # Default to empty, will be set later based on chain_id or user input
 chain_id="akashnet-2"  # Default chain ID
 provider_version=""  # Will be fetched from Helm Chart
 node_version=""  # Will be fetched from Helm Chart
 install_gpu_support=false
 gpu_nodes=()
-install_storage_support=false
-use_pricing_script=false
-storage_class_name=""
+install_storage_support=true
+use_pricing_script=true
+storage_class_name="beta3"
+
+### LOGIC THAT DETERMINES EPHEMERAL STORAGE LOCATION
+
+# Default paths
+DEFAULT_KUBELET_DIR="/var/lib/kubelet"
+DEFAULT_CONTAINERD_DIR="/var/lib/containerd"
```

Comment:
This is the IMAGEFS directory.
Which is /var/lib/containerd for kubespray based deployments.
/var/lib/rancher/k3s/agent/containerd - for K3S based deployments.